src/content/docs/new-relic-control/agent-control/configuration.mdx

```diff
@@ -0,0 +1,284 @@
+                  # Ref: `https://github.com/newrelic/helm-charts/tree/master/charts/nri-bundle%60 
+                  # Recommended: check and define an explicit chart version (latest stable)
+                 chart_version: "*"
+             agent-operator:
+               type: newrelic/com.newrelic.k8s_agent_operator:0.1.0
+               content:
+                 chart_version: "*"
+             fluentbit:
+               type: newrelic/io.fluentbit:0.1.0
+               content:
+                  # Ref: `https://github.com/newrelic/helm-charts/tree/master/charts/newrelic-logging`
+                  # Recommended: check and define an explicit chart version (latest stable)
+                 chart_version: "*"
+                 chart_values:
+                   global:
+                     lowDataMode: true
+             prometheus:
+               type: newrelic/com.newrelic.prometheus:0.1.0
+               content:
+                 chart_version: "*"
+                 chart_values:
+                   global:
+                     lowDataMode: true
+                   newrelic-prometheus-agent:
+                     config.kubernetes.integrations_filter.enabled: false
+    ```
+
+  </Collapser>
+
+</CollapserGroup>
+
+### Remote configuration: New Relic infrastructure
+
+
+<CollapserGroup>
+  <Collapser
+    id="infra-remote-config"
+    title="Infrastructure remote configuration"
+  >
+
+    ```yaml
+    chart_version: "*"
+      chart_values:
+        newrelic-infrastructure:
+          enableProcessMetrics: true
+    ```
+
+  </Collapser>
+
+</CollapserGroup>
+
+### Remote configuration: Fluent Bit
+
+
+<CollapserGroup>
+  <Collapser
+    id="fluentbit-remote-config"
+    title="Fluent Bit configuration"
+  >
+
+    ```yaml
+      chart_version: "*"
+      chart_values:
+        newrelic-logging:
+          sendMetrics: true
+    ```
+
+  </Collapser>
+
+</CollapserGroup>
+
+### Remote configuration: Prometheus
+
+
+<CollapserGroup>
+  <Collapser
+    id="prometheus-config"
+    title="Prometheus configuration"
+  >
+
+    ```yaml
+      chart_version: "*"
+      chart_values:
+        newrelic-prometheus-agent:
+          lowDataMode: true
+    ```
+
+  </Collapser>
+
+</CollapserGroup>
+
+### Remote configuration: OpenTelemetry
+
+
+<CollapserGroup>
+  <Collapser
+    id="prometheus-config"
+    title="Prometheus configuration"
+  >
+
+    <Callout variant="important">
+      Create a Kubernetes secret to securely store the New Relic license key and use it in the `chart_values` in replacement of the `licenseKey` value:
+
+          ```yaml
+              customSecretName: "your-secret-name"
+                customSecretLicenseKey: "your-secret-key"
+          ```
+    </Callout>
+
+    For this approach, create a configuration in Fleet Control UI with the following structure, specifying the desired `receivers.filelog.enabled` value and referencing any other relevant Helm chart values:
+            
+          ```yaml
```

Comment:
I think that all yaml are broken since in markdown it ignores the first spaces, but then it adds them... therefore it seems valid but it is like 
<img width="446" alt="Screenshot 2025-02-19 at 18 06 19" src="https://github.com/user-attachments/assets/3f202bc4-ccf6-410e-bcfb-2cef95566470" />